definitions/tools/cnvkit.cwl

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/usr/bin/python", "/usr/local/bin/cnvkit.py", "batch"]
+
+requirements:
+    - class: ShellCommandRequirement
+    - class: DockerRequirement
+      dockerPull: "etal/cnvkit"
+
+inputs:
+    tumor_bam:
+        type: File
+        inputBinding:
+            position: 1
+    normal_bam:
+        type: File
+        inputBinding:
+            position: 2
+            prefix: "--normal"
+    bait_intervals:
+        type: File
+        inputBinding:
+            position: 3
+            prefix: "--targets"
+    reference:
+        type: File
+        inputBinding:
+            position: 4
+            prefix: "--fasta"
+    access:
+        type: File?
+        inputBinding:
+            position: 5
+            prefix: "--access"
+    method:
+        type: string?
+        inputBinding:
+            position: 6
+            prefix: "--method"
+            default: "hybrid"
```

Comment:
I believe the `default` goes one level higher rather than inside the `inputBinding`.

Reply:
Fixed!